DistributionPackages/Neos.NeosIo/Classes/Command/CrowdCommandController.php

```diff
@@ -164,7 +164,7 @@ public function setUserAttributeCommand($userName, $attribute, $value)
      *
      * Structure has to similiar to this:
      *
-     * username;neos_googleplus;neos_email;neos_twitter;...
+     * username;neos_email;neos_twitter;...
      * xyz;xyz;xyz@example.org;@xyz;...
```

Comment:
```suggestion
     * xyz;xyz@example.org;@xyz;...
```

Reply:
I don't think it matters. Actually we could just remove the command as by now it's easier to change the values in crowd directly which was not possible at the time.

Reply:
I could also split it up. First intention was to remove the button on the page and then I found more and more google plus stuff ;)